unit_tests/testGeom_CircCyl/testGeom_CircCyl.f90

```diff
@@ -102,6 +110,7 @@ SUBROUTINE TestCircle_and_Cylinder
       ASSERT(bool, 'circle1%intersectLine(...) (tangent)')
       CALL line1%clear()
 
+
```

Comment:
Lets fix this. No need to add extra whitespace.

I ran all the heavies under MPACT. 2q failed, but it appeared to be spurious. I reran the test by itself through ctest and it passed. 2q is the case with the grid spacer.